Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM python:3.9-alpine as base
+
+# Set environment variable
+ENV POETRY_HOME="/opt/poetry"
+ENV PATH="$POETRY_HOME/bin:$PATH"
+
+# Install poetry
+RUN pip install poetry Flask gunicorn
+
+# Set up the working directory
+WORKDIR /app
+
+# Copy the content of the repo inside the image
+COPY . /app
+
+
+
+# Dependecies stage
+FROM base as dependencies
+
+# Install dependecies withtout the dev dependencies
+RUN poetry install --no-dev
+
+
+# Development stage
+FROM dependencies as development_env
+
+# Install the development dependencies
+RUN poetry install 
+
+# Expose the Flask development port
+EXPOSE 5000
+
+
+# Production environment
+FROM base as production
+
+#copy the dependencies
+COPY --from=dependencies /usr/local/lib/python3.9/site-packages /usr/local/lib/python3.9/site-packages
+
+#copy the application code from the dependencies stage
+COPY --from=dependencies /app /app 
```

Comment:
I think this is basically irrelevant, since this folder shouldn't have changed between base and dependencies, so it's already set up (since we're already building off base) - but definitely the case if we switch to building from `dependencies `